### PR TITLE
Models

### PR DESCRIPTION
**Pull request by anhphamdt98**
## Related Tickets:
- [#27399](https://edu-redmine.sun-asterisk.vn/issues/27399)

## WHAT this PR does?
- Create Eloquent models and create relationships between them

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by anhphamdt98** [comment]
ready

File: app/Models/Comment.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+{
+    protected $fillable = [
+        'user_id',
+        'post_id',
+        'comment',
+    ];
+
+    public function user() 
+    {
+        return $this->belongsTo(User::Class);
```
Comment (huyenttm-0901) [inline app/Models/Comment.php]:
```suggestion
        return $this->belongsTo(User::class);
```
sửa hết đi em

**Comment by anhphamdt98** [comment]
ready

File: app/Models/User.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    }
+
+    public function following()
+    {
+        return $this->belongsToMany(User::class, 'user_and_followers', 'follower_id', 'user_id');
+    }
+
+    public function followers()
+    {
+        return $this->belongsToMany(User::class, 'user_and_followers', 'user_id', 'follower_id');
```
Comment (huyenttm-0901) [inline app/Models/User.php]: sửa lại tên bảng

**Comment by anhphamdt98** [comment]
ready

**Comment by anhphamdt98** [comment]
ready

**Pull request merged**